docs/dev-tools/zombienet/tutorials/minimal-example.md

```diff
@@ -0,0 +1,235 @@
+            <td>tail -f /var/folders/f4/7rdt2m9d7j361dm453cpggbm0000gn/T/zombie-75a01b93c92d571f6198a67bcb380fcd_21724-2</td>
+        </tr>
+        <tr>
+            <th>Parachain ID</th>
+            <td>100</td>
+        </tr>
+        <tr>
+            <th>ChainSpec Path</th>
+            <td>/var/folders/f4/7rdt2m9d7j361dm453cpggbm0000gn/T/zombie-75a01b93c92d571f6198a67bcb380fcd_21724-2</td>
+        </tr>
+    </table>
+</div>
+
+!!! note 
+    If the IPs and ports are not explicitly defined in the configuration file, they may change each time the network is started, causing the links provided in the output to differ from the example.
+
+## Interacting with the Spawned Network
+
+### Connecting to the Nodes
+
+After the network is launched, you can interact with it using the [Polkadot.js App](https://polkadot.js.org/apps/){target=_blank}. To do so, open your browser and use the provided links listed by the output as 'direct link'. For instance, in this particular case, as the ports may vary from spawning to spawning, to interact with the `alice` node, open [https://polkadot.js.org/apps/?rpc=ws://127.0.0.1:55308#explorer](https://polkadot.js.org/apps/?rpc=ws://127.0.0.1:55308#explorer){target=_blank} as it is the link provided in the output for the `alice` node. Moreover, you can also do this for the `bob` and `collator01` nodes.
+
+If you want to interact with the nodes more programmatically, you can also use the [Polkadot.js API](https://polkadot.js.org/api/){target=_blank}. For example, the following code snippet shows how to connect to the `alice` node using the Polkadot.js API and log some information about the chain and node:
+
+```typescript
+import { ApiPromise, WsProvider } from '@polkadot/api';
+
+async function main() {
+  const wsProvider = new WsProvider('ws://127.0.0.1:55308');
+    const api = await ApiPromise.create({ provider: wsProvider });
+
+    // Retrieve the chain & node information via rpc calls
+    const [chain, nodeName, nodeVersion] = await Promise.all([
+        api.rpc.system.chain(),
+        api.rpc.system.name(),
+        api.rpc.system.version()
+    ]);
+
+    console.log(`You are connected to chain ${chain} using ${nodeName} v${nodeVersion}`);
+}
+
+main().catch(console.error).finally(() => process.exit());
+```
+
+Either way allows you to interact easily with the network and its nodes.
+
+### Checking Metrics
+
+You can also check the metrics of the nodes by accessing the provided links listed by the output as `Prometheus Link`. [Prometheus](https://prometheus.io/){target=_blank} is a monitoring and alerting toolkit that collects metrics from the nodes. By accessing the provided links, you can see the metrics of the nodes in a web interface. So, for example, the following image shows the Prometheus metrics for Bob’s node from the Zombienet test:
+
+![Prometheus metrics for Bob’s node from the Zombienet test](/polkadot-ecosystem-docs-draft/images/dev-tools/zombienet/tutorials/zombienet-tutorials-1.webp)
```

Comment:
This is kinda small, might need to zoom in a bit more and retake it. Is this a portion of the screen? Will users know what this is when they see it or how to navigate to it?

Reply:
@eshaben 
This is the view you will see in your browser when you open `Prometheus Link.` The window is long and displays plain metrics. The image is already zoomed in at 175%. Should I zoom in further?

Reply:
Actually, can you just add in the browser window (a dark-themed window like the ones on the [moonbeam docs](https://docs.moonbeam.network/builders/ethereum/dev-env/remix/#add-a-smart-contract-to-the-file-explorer))? It just seems like it'll provide more context, as this looks like it could be terminal output

Wow already at 175? Yeah, I would zoom in a little bit more so it's a bit more readable

Reply:
Oh, okay, I thought that we intended to hide the URL when taking screenshots. Will do that and also apply a bit more zoom here :)